src/main/java/net/minecraftforge/common/IAnvilRecipe.java

```diff
@@ -0,0 +1,56 @@
+package net.minecraftforge.common;
+
+import java.util.List;
+
+import net.minecraft.item.ItemStack;
+import net.minecraft.world.World;
+
+public interface IAnvilRecipe
+{
+    /**
+     * Returns the input for this recipe, any mod accessing this value should never
+     * manipulate the values in this array as it will effect the recipe itself.
+     */
+    List[] getInputs();
```

Comment:
It would be better for me to change this completely so it was more like the crafting table recipes, and return a ArrayList<Object> here instead.

Also noticed while I did replace all tabs with spaces I forgot to put the open brackets on new lines.

I'll get this fixed up.